TLM/TLM/Manager/AbstractGeometryObservingManager.cs

```diff
@@ -63,14 +64,14 @@ public abstract class AbstractGeometryObservingManager : AbstractCustomManager,
                 // Handle a segment update
                 ExtSegment seg = (ExtSegment)update.segment;
                 if (!seg.valid) {
-#if DEBUGGEO
-                    if (GlobalConfig.Instance.Debug.Switches[5])
+#if DEBUG
+                    if (DebugSwitch.GeometryDebug.Get())
```

Comment:
Can these be converted to `Log._DebugIf()` calls with the string formatters in lambda functions?

Reply:
They can. This file is ahead, i have stopped after doing CSUtil.Commons and Custom AI code. Will continue from there in another branch.